src/test/java/org/folio/innreach/domain/listener/KafkaCirculationEventListenerApiTest.java

```diff
@@ -50,6 +56,9 @@
   private static final Duration ASYNC_AWAIT_TIMEOUT = Duration.ofSeconds(15);
   private static final Date DUE_DATE = new Date();
 
+  @MockBean
+  private InnReachExternalService innReachMockExternalService;
```

Comment:
There is already `innReachExternalService` defined, replace its annotation to `@MockBean` instead

Reply:
Fixed